actions/admin_actions.jsx

```diff
@@ -347,3 +347,13 @@ export function unregisterAdminConsolePlugin(pluginId) {
         });
     };
 }
+
+export async function testSiteURL(success, error, config) {
+    const siteURL = config['ServiceSettings.SiteURL'];
```

Comment:
i think the original suggestion was to pass the siteURL directly, i think that would be the way, otherwise this function becomes not very re-usable.